frontend/pages/DetailedDataset/[id].js

```diff
@@ -294,7 +294,7 @@ export default function DetailedDataset({ data, uri, prevUserId, prevLoggedUsern
                 : 'Samordnet'}
             </div>
           )}
-        </p>
```

Comment:
Having div inside p gives error in console, fixed with changing p to div, did not effect functionality or styling.